src/js/views/forms/form/form_views.js

```diff
@@ -169,12 +176,31 @@ const LayoutView = View.extend({
       replaceElement: false,
     },
     status: '[data-status-region]',
+    widgets: '[data-widgets-region]',
+  },
+  templateContext() {
+    return {
+      formHasWidgets: this.formHasWidgets,
```

Comment:
I think we should move the logic from initialize to here for the widgets even though it doesn't change, I think it's good practice.  Also I believe at this point widgets is always defined and will have a length.  So I suspect that the `widget &&` is a missing else branch in the coverage tests.